google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/AppEngineCloudConfigurable.java

```diff
@@ -50,16 +52,25 @@
 
   private final AppEngineServerConfiguration configuration;
   private final SystemEnvironmentProvider environmentProvider;
+  private static final String MORE_INFO_URI_OPEN_TAG = "<a href='https://cloud.google.com/appengine/docs/flexible/'>";
```

Comment:
Unfortunately not. Markup and urls should be avoided in localization bundles, since translators will have access to modify them.